src/overlays/actors/ovl_Obj_Warp2block/z_obj_warp2block.c

```diff
@@ -43,36 +43,286 @@ const ActorInit Obj_Warp2block_InitVars = {
     (ActorFunc)ObjWarp2block_Update,
     (ActorFunc)ObjWarp2block_Draw,
 };
-*/
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Warp2block/func_80BA1DF0.s")
+typedef struct {
+  f32 scale;
+  f32 focus;
+  s16 params;
+} Warp2BlockSpawnData;
+
+ Warp2BlockSpawnData sSpawnData[2] = {
+    { 1.0f, 60.0f, 0x0018 },
```

Comment:
you can omit the size specification. in the event someone wants to add or remove entries in the future for modding purposes, this makes it easier

Reply:
Done.